src/test/test-utils.ts

```diff
@@ -0,0 +1,26 @@
+import { WranglerActionConfig } from "../wranglerAction";
+
+export function getTestConfig({
```

Comment:
testConfig didn't immediately hit in my mind as a test version of WranglerActionConfig when I saw it used in the test file. How do you feel about naming it more specifically testWranglerActionConfig? I know it's longer and that's basically the name of the repo, so I'm open to leaving it as-is if you feel strongly about it.

Reply:
I honestly kinda like the shorter name! I think there's no other configs in the repo, it's probably fine?

Reply:
yeah I think since we only have one get config function it's ok